docs/swagger/traPortfolio.v1.yaml

```
@@ -1190,9 +1190,9 @@ components:
         duration:
           format: date-time
           description: |-
-            プロジェクト期間の配列
+            プロジェクト期間のオブジェクト
             untilがなかった場合存続中
-          type: array
+          type: object
           items:
             $ref: '#/components/schemas/ProjectDuration'
```

```suggestion
          $ref: '#/components/schemas/ProjectDuration'
```
オブジェクトにしたいならこうですね

https://swagger.io/docs/specification/data-models/data-types/#object
https://swagger.io/docs/specification/using-ref/

---

なるほど:kansya: